Trading/Exchange/binance/binance_exchange.py

```diff
@@ -90,6 +90,42 @@ def _fill_order_missing_data(self, order, trades):
         if not order[trading_enums.ExchangeConstantsOrderColumns.FEE.value] and order_id in trades:
             order[trading_enums.ExchangeConstantsOrderColumns.FEE.value] = \
                 trades[order_id][trading_enums.ExchangeConstantsOrderColumns.FEE.value]
+                
+    async def _create_market_stop_loss_order(
```

Comment:
It is only concerning stop market?

Reply:
For now just stop market, stop_limit needs to be implemented, but its supported for all pairs as far as I found out